dds/idl/be_init.cpp

```diff
@@ -7,13 +7,16 @@
 
 #include "../Version.h"
 
-#include "tao/Version.h"
-#include "global_extern.h"
+#include <ace/OS_NS_stdlib.h>
```

Comment:
Does it need to be first?  Usual order would be dds, tao, ace, stdlib.

Reply:
I wasn't aware there was a usual order. My preference, although I didn't really follow it here, to do companion header (if applicable source file), then stdlib, 3rd party libs (ace then tao), and finally local headers. I'm willing to follow that order though if it's the convention. Should the order you're talking about be in the guidelines?

Reply:
I wasn't aware there was a usual order. My preference, although I didn't really follow it here, to do companion header (if applicable source file), then stdlib, 3rd party libs (ace then tao), and finally local headers. I'm willing to follow that order though if it's the convention. Should the order you're talking about be in the guidelines?

Reply:
As to what the usual order is (and why), there's some good background info here:
https://stackoverflow.com/a/14243688

Reply:
That makes sense. This might have prevented some of the trouble I had with https://github.com/objectcomputing/OpenDDS/pull/1339/commits/7424d65e550153435296fdcee9e3b85912659fba. I've included this rule in #1398 and will definitely try to follow it in the future.